python/tests/scripts/d3_deptypes.metta

```diff
@@ -1,55 +1,75 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ; A classical example of dependent types
-
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Simple type definitions:
+; Defines the natural numbers `Nat` for counting
 (: Nat Type)
 (: Z Nat)
 (: S (-> Nat Nat))
 
+; `Vec` is defined as a dependent type: it takes a type ``$t` and a `Nat` length.
```

Comment:
Excessive `. Also, "it takes a `$t` type and a `Nat` length" sounds more uniform.